apps/dashboard/app/javascript/packs/files/data_table.js

```diff
@@ -198,11 +194,6 @@ class DataTable {
                     data: null,
                     orderable: false,
                     defaultContent: '<input type="checkbox">',
-                    render: function (data, type, row, meta) {
-                        var api = new $.fn.dataTable.Api(meta.settings);
-                        let selected = api.rows(meta.row, { selected: true }).count() > 0;
-                        return `<input type="checkbox" ${selected ? 'checked' : ''}> ${selected ? 'checked' : ''}`;
-                    }
```

Comment:
I don't believe this is ever used or is useful. plus it adds the text `checked` to the innerHtml?